src/huggingface_hub/keras_mixin.py

```diff
@@ -178,6 +311,7 @@ def push_to_hub_keras(
         include_optimizer=include_optimizer,
         **model_save_kwargs,
     )
+    _create_model_card(model, repo_path_or_name, plot_model, task_name)
```

Comment:
Something sneaky here...if you call this within `push_to_hub_keras`, it won't get executed for the `KerasModelHubMixin` (for custom Keras components). If you instead stick this within the `save_pretrained_keras` function, it'll happen for the mixin as well (because the mixin calls explicitly to that).

Reply:
Sure I'll do it! Thanks for the catch.

Reply:
I have a problem with this. ModelHubMixin class has `_save_pretrained` that only takes file path, and you can extend it with `**model_save_kwargs` during overriding for Keras mixin, but I also need to pass model card related `kwargs`, like `plot_model` and `task_name` otherwise I need to override the parent class to take two model card related kwargs. I think it's an overkill to extend parent class like that. What do you suggest? I added those arguments inside `save_pretrained_keras` but for custom components I need `model.save_pretrained`, no?

Reply:
Afaik nobody is using the mixin atm btw :smile:

Reply:
@osanseviero I already did it for `save_pretrained_keras` and it works. I can push that.